features/out_of_memory.feature

```diff
@@ -20,6 +20,29 @@ Feature: Reporting out of memory events
         And the event "app.bundleVersion" equals "5"
         And the event breadcrumbs contain "Crumb left before crash"
 
+    @ci_skip
+    Scenario: The OS kills the application while inactive
+        The application is in the foreground but inactive when interrupted 
+        by a phone call or Siri
+
+        When I run "OOMScenario"
+        And the app is interrupted by Siri
+        And the app is unexpectedly terminated
+        And I relaunch the app
+        And I wait for 10 seconds
+        Then I should receive 0 requests
+
+    Scenario: The app is terminated normally
+        The application can be gracefully terminated by the OS if more
+        memory is needed for other applications or directly by calling
+        exit(0)
+
+        When I run "OOMWillTerminateScenario"
+        And I wait for 4 seconds
+        And I relaunch the app
```

Comment:
I might be reading this wrong, but it appears as though the `relaunch the app` step in mazerunner launches the fixture with the same `EVENT_TYPE`. If that's true, it would mean that in this scenario the app always terminates and zero requests would be expected regardless of OOM detection.